### PR TITLE
Add `--use-cuda-base-image` flag

### PR DESCRIPTION
**Pull request by mattt**
Alternative to #1205

### CODE REVIEW

File: test-integration/test_integration/test_predict.py [hunk 1]
```diff
@@ -58,7 +58,9 @@ def test_predict_writes_files_to_files(tmpdir_factory):
         capture_output=True,
     )
     assert result.stdout == b""
-    with open(out_dir / "output.bmp", "rb") as f:
+    # something about this PR makes the default output be "output" instead of "output.bmp"
+    with open(out_dir / "output", "rb") as f:
+    # with open(out_dir / "output.bmp", "rb") as f:
```
Comment (hongchaodeng) [inline test-integration/test_integration/test_predict.py:63]: remove commented code?

**Review by hongchaodeng — APPROVED** [review]
lgtm after nit!

File: test-integration/test_integration/test_predict.py (hunk 1 above)
Comment (technillogue) [reply test-integration/test_integration/test_predict.py:63]: do you have any idea what might be causing this error?

Comment (technillogue) [reply test-integration/test_integration/test_predict.py]: it was getting reported as x-ms-bmp instead of bmp in python:3.8-slim for some reason

**Comment by mattt** [comment]
Nice work, @technillogue. LGTM!

**Pull request merged**